test/Maestro.ScenarioTests/ScenarioTests_GitHubFlow.cs

```diff
@@ -87,7 +87,7 @@ public ScenarioTests_GitHubFlow()
             new DependencyDetail
             {
                 Name = "Foo",
-                Version = "1.1.0",
+                Version = "1.17.0",
                 RepoUri = sourceRepoUri,
                 Commit = TestRepository.CoherencyTestRepo1Commit,
```

Comment:
The commits won't match, we're using CoherencyTestRepo2Commit when we're setting up the build here https://github.com/dotnet/arcade-services/blob/a34c4514c8c006bb1d12e04a250dbd36af079f03/test/Maestro.ScenarioTests/EndToEndFlowLogic.cs#L322

Reply:
```suggestion
                Commit = TestRepository.CoherencyTestRepo2Commit,
```